itkzlib/zlib.h

```diff
@@ -1085,7 +1085,7 @@ ZEXTERN gzFile ZEXPORT gzopen  OF((const char *path, const char *mode));
 ZEXTERN gzFile ZEXPORT gzdopen  OF((int fd, const char *mode));
 /*
      gzdopen() associates a gzFile with the file descriptor fd.  File
-   descriptors are obtained from calls like open, dup, creat, pipe or
+   descriptors are obtained from calls like open, dup, create, pipe or
```

Comment:
please review

Reply:
This is supposed to be `creat`:

  https://www.zlib.net/manual.html